example/ios/example/AppDelegate.mm

```diff
@@ -10,6 +10,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   // You can add your custom initial props in the dictionary below.
   // They will be passed down to the ViewController used by React Native.
   self.initialProps = @{};
+  [GMSServices provideAPIKey:@"YOUR_GOOGLE_API_KEY"]; // add this line using the api key obtained from Google Console
```

Comment:
Onde se lle pasan o user_id e o api_key?, penso que para o usuario sería máis cómodo que estivesen xuntos

Reply:
Si, sería máis cómodo que estivera todo xunto, pero hai que andar pasando información dende o lado TS ao nativo.
Este requisito forma parte do "how to install" de RN Google Maps, quizais poidamos tirar ahora mismo así, pero como pouco documentar.

Reply:
Documentado en `example/README.md`